exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,29 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+const mentor1 = "Daniel";
+const mentor2 = "Irina";
+const mentor3 = "Mimi";
+const mentor4 = "Rob";
+const mentor5 = "Yohannes";
+
+function shoutyGreeting(mentors) {
+  mentors = mentors.toUpperCase();
+  console.log("HELLO " + mentors);
+}
+
+shoutyGreeting(mentor1);
+shoutyGreeting(mentor2);
+shoutyGreeting(mentor3);
+shoutyGreeting(mentor4);
+shoutyGreeting(mentor5);
+
+/*const shoutyGreeting1 = createGreeting(mentor1);
+const shoutyGreeting2 = createGreeting(mentor2);
+const shoutyGreeting3 = createGreeting(mentor3);
+const shoutyGreeting4 = createGreeting(mentor4);
+const shoutyGreeting5 = createGreeting(mentor5);
+
+console.log(shoutyGreeting1);
+console.log(shoutyGreeting2);
+console.log(shoutyGreeting3);
+console.log(shoutyGreeting4);
+console.log(shoutyGreeting5);
+*/
```

Comment:
It is good to get into the habit of deleting any experimental code that you have commented out.

Reply:
I was thinking about deleting it so it doesn't look cluttered but decided to leave it so I would remember the different ways that the exercise could solved.